Sensory/src/main/AndroidManifest.xml

```diff
@@ -6,12 +6,12 @@
     />
   <uses-permission
         android:name="android.permission.WRITE_EXTERNAL_STORAGE"
-        android:maxSdkVersion="29"
+        android:maxSdkVersion="32"
```

Comment:
Why do you need the max SDK version here?

Reply:
You're right. We might not need it. 

The IDE strangely suggests to set maxSdkVersion to 32, and then again gives a warning.
First warning: `WRITE_EXTERNAL_STORAGE is deprecated (and is not granted) when targeting Android 13+. If you need to write to shared storage, use the`
Second warning: `WRITE_EXTERNAL_STORAGE no longer provides write access when targeting Android 11+, even when using requestLegacyExternalStorage`